docs/runbooks/GOOGLE-ANALYTICS.md

```diff
@@ -0,0 +1,76 @@
+# Google Analytics
+
+## Event Tracking
+We use Google Analytics to track events within the app to highlight patterns in user behaviour. This can be used to analyse or evaluate a change made to a page, for example:
+
+ - Do additional form fields result in a drop in the amount of users submitting a form?
+ - Has changes to the style of a button resulted in more people clicking it?
+ - Do users scroll all the way to the bottom of the page or just click the continue button?
+
+ Google Analytics accepts a number of data points related to an event:
+
+ #### Category (required)
+ Describes the page, module or component the event belongs to.
```

Comment:
Think we should give examples, maybe even a table of the expected categories in DigiDeps by current pages to give a feel for how this will work? 

Looking at the convention from the post you reference, I wonder if we could make it a bit cleaner.

Category: {Page Title}:{Sub Section i.e. in a form, optional}
Action: {Event }:{Element Type}
Label: {Human summary and additional detail}

Category: Finances : Outgoings
Action: Click : Help Link
Label: Clicked link to help page  /help